app/main.py

```diff
@@ -15,5 +15,21 @@
     "Sergio": 3,
 }
 collection_of_coins = {1, 2, 25}
-
-# write your code here
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
+all_types = [lucky_number,
+             pi,
+             one_is_a_prime_number,
+             name,
+             my_favourite_films,
+             profile_info,
+             marks,
+             collection_of_coins]
+for typee in all_types:
```

Comment:
Consider renaming the variable `typee` to something more descriptive, like `item` or `element`, to improve code readability.